fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PipeAnalyzer.java

```diff
@@ -80,6 +80,10 @@ public static void analyzePipeName(PipeName pipeName, ConnectContext context) {
         FeNameFormat.checkCommonName("pipe", pipeName.getPipeName());
```

Comment:
Why not use `FeNameFormat.checkDbName(pipeName.getDbName());`

Reply:
Nice catch. Updated.